src/Raven.Server/Web/System/OngoingTasksHandler.cs

```diff
@@ -21,12 +21,15 @@
 
 namespace Raven.Server.Web.System
 {
-    public class OngoingTasksHandler : RequestHandler
+    public class OngoingTasksHandler : DatabasesHandler
     {
-        [RavenAction("/admin/ongoing-tasks", "GET", "/admin/ongoing-tasks?databaseName={databaseName:string}")]
+        [RavenAction("ongoing-tasks", "GET", AuthorizationStatus.ValidUser)]
```

Comment:
This need to be double checked for permissions, I _think_ that viewing is fine, but need to check